app/controllers/ctc/questions/remove_spouse_controller.rb

```diff
@@ -0,0 +1,33 @@
+module Ctc
+  module Questions
+    class RemoveSpouseController < QuestionsController
+      include AuthenticatedCtcClientConcern
+      include AnonymousIntakeConcern
+
+      layout "intake"
```

Comment:
Do all the controllers in the CTC flow use the same layout? Could this be inherited?